routes/login.js

```diff
@@ -1,25 +1,24 @@
-//const Joi = require('joi');
-const app = require("express");
+const app = require('express');
+
 const router = app.Router();
-const bcrypt = require("bcrypt");
-const User = require("../models/user");
-const { UserBodyLoginSchema } = require("./validators/validators");
-const validator = require("express-joi-validator");
+const User = require('../models/user');
+const { UserBodyLoginSchema } = require('./validators/validators');
+const validator = require('express-joi-validator');
 
-router.post("/", validator(UserBodyLoginSchema), (req, res, next) => {
-  //console.log(req.body);
-  User.authenticate(req.body.email, req.body.password, function(error, user) {
-    console.log(error);
-    if (error || !user) {
-      const err = new Error("Wrong email or password.");
+router.post('/', validator(UserBodyLoginSchema), (req, res, next) => {
+  User.authenticate(req.body.email, req.body.password, (error, user) => {
+    if (error) {
+      const err = new Error('Error! Wrong email or password.');
+      err.status = 401;
+      return next(error);
+    } else if (!user) {
+      const err = new Error('User! Wrong email or password.');
       err.status = 401;
       return next(err);
-    } else {
-      req.session.userId = user._id;
-      res.cookie("simul-doc", user._id, { signed: true, httpOnly: true });
-      res.send({ message: "User is authenticated." });
-      //return res.redirect("/profile");
     }
+    req.session.userId = user._id;
+    res.cookie('simul-doc', user._id, { signed: true, httpOnly: true });
+    return res.send({ message: 'User is authenticated.' });
```

Comment:
Instead of this message you can send some user profile data like e-mail and other useful stuff

Reply:
Done